cumulusci/tasks/bulkdata.py

```diff
@@ -142,107 +199,55 @@ class LoadData(BaseSalesforceApiTask):
             "description": "The path to a yaml file containing mappings of the database fields to Salesforce object fields",
             "required": True,
         },
+        "start_step": {
```

Comment:
doesn't block merge: as we develop more tasks that have their internal control flow I'd like to make sure we do it consistently

Reply:
> Sure sounds like tasks are a kind of flow ;-)

Sure does. This...update dependencies... they both have substeps that we want to give progress on....